moped-database/migrations/1667514062398_set_fk_public_moped_project_project_lead_id/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."moped_project"
+  add constraint "moped_project_project_lead_id_fkey"
+  foreign key ("project_lead_id")
+  references "public"."moped_entity"
+  ("entity_id") on update cascade on delete set null;
```

Comment:
i would lean toward squashing this into one migration, since the feature requires both of them.

Reply:
done!